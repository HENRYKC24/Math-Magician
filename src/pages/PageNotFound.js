import React from 'react';

const PageNotFound = () => (
  <div className="not-found">
    <h1>Page Not Found!</h1>
  </div>
);

export default PageNotFound;
