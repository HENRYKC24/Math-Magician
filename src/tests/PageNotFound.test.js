import React from 'react';
import renderer from 'react-test-renderer';
import PageNotFound from '../pages/PageNotFound';

it('PageNotFound page renders the UI as expected', () => {
  const tree = renderer.create(<PageNotFound />).toJSON();
  expect(tree).toMatchSnapshot();
});
