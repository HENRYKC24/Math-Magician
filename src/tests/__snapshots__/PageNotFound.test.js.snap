// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound page renders the UI as expected 1`] = `
<div
  className="not-found"
>
  <h1>
    Page Not Found!
  </h1>
</div>
`;
